services/terraformer/server/loadbalancer/loadbalancer.go

```diff
@@ -72,27 +72,27 @@ func (l LBcluster) Build() error {
 
 func (l LBcluster) Destroy() error {
 	cluster := clusterBuilder.ClusterBuilder{
-		//DesiredInfo: , //desired state is not used in DestroyNodepools
+		//DesiredInfo: desired state is not used in Destroy
 		CurrentInfo: l.CurrentLB.ClusterInfo,
 		ProjectName: l.ProjectName,
-		ClusterType: pb.ClusterType_LB}
-	nodeIPs := getNodeIPs(l.CurrentLB.ClusterInfo.NodePools)
+		ClusterType: pb.ClusterType_LB,
+	}
+	if err := cluster.DestroyNodepools(); err != nil {
+		return fmt.Errorf("error while destroying the K8s cluster %s : %w", l.CurrentLB.ClusterInfo.Name, err)
```

Comment:
`error while destroying the K8s cluster` ->`error while destroying the LB cluster`

A typo from ages ago